tests/vxlan/test_vxlan_ecmp.py

```diff
@@ -1970,3 +1970,425 @@ def test_vxlan_varying_src_ip(self, setUp, encap_type):
+                netid=NEXTHOP_PREFIX))
+
+        Logger.info("Create a new destination")
+        tc1_new_dest = ecmp_utils.get_ip_address(
+            af=ecmp_utils.get_payload_version(encap_type),
+            netid=DESTINATION_PREFIX)
+
+        Logger.info("Map the new destination and the new endpoint(s).")
+        self.setup[encap_type]['dest_to_nh_map'][vnet][tc1_new_dest] = \
+            tc1_end_point_list
+
+        Logger.info("Create a new priority endpoint config and Copy to the DUT.")
+
+        Logger.info("Create the json and apply the config in the DUT swss.")
+        # The config looks like:
+        # [
+        #   {
+        #     "VNET_ROUTE_TUNNEL_TABLE:vnet:tcx_new_dest/32": {
+        #       "endpoint": "{tcx_end_point_list}"
+        #       "endpoint_monitor": "{tcx_end_point_list}",
+        #       "primary" : "{}",
+        #       "adv_prefix" : "{}/{}",
+        #     },
+        #     "OP": "{}"
+        #   }
+        # ]
+        try:
+            ecmp_utils.create_and_apply_priority_config(
+                self.setup['duthost'],
+                vnet,
+                tc1_new_dest,
+                ecmp_utils.HOST_MASK[ecmp_utils.get_payload_version(encap_type)],
+                tc1_end_point_list,
+                [tc1_end_point_list[0]],
+                "SET")
+            # both primary secondary are up.
+            # only primary should recieve traffic.
+            time.sleep(2)
+            down_list = tc1_end_point_list[1]
+            if isinstance(down_list, str):
+                down_list = [down_list]
+            self.setup['list_of_downed_endpoints'] = set(down_list)
+            # setting both primary and secondary as up. only primary will recieve traffic.
+            ecmp_utils.set_vnet_monitor_state(self.setup['duthost'],
+                                              tc1_new_dest,
+                                              ecmp_utils.HOST_MASK[ecmp_utils.get_payload_version(encap_type)],
+                                              tc1_end_point_list[0], "up")
+            ecmp_utils.set_vnet_monitor_state(self.setup['duthost'],
+                                              tc1_new_dest,
+                                              ecmp_utils.HOST_MASK[ecmp_utils.get_payload_version(encap_type)],
+                                              tc1_end_point_list[1], "up")
+            time.sleep(10)
+            # verifying overlay_dmac
+            result = \
+                self.setup['duthost'].shell("sonic-db-cli APPL_DB HGET 'VNET_MONITOR_TABLE:{}:{}/{}' 'overlay_dmac'"
+                                            .format(
+                                                tc1_end_point_list[0],
+                                                tc1_new_dest,
+                                                ecmp_utils.HOST_MASK[ecmp_utils.get_payload_version(encap_type)]))
+            assert result['stdout'] == u'25:35:45:55:65:75'
```

Comment:
can you define this as a variable instead of hardcode?

Reply:
done